homeassistant/components/webostv/config_flow.py

```diff
@@ -0,0 +1,225 @@
+"""Config flow to configure webostv component."""
+import functools
+import json
+import logging
+from urllib.parse import urlparse
+
+from aiopylgtv import PyLGTVPairException
+from getmac import get_mac_address
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_CLIENT_SECRET, CONF_HOST, CONF_ICON, CONF_NAME
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv
+from homeassistant.util.network import is_ip_address
+
+from . import CannotConnect, async_control_connect
+from .const import (
+    CONF_ON_ACTION,
+    CONF_SOURCES,
+    DEFAULT_NAME,
+    DEFAULT_SOURCES,
+    DOMAIN,
+    TURN_ON_DATA,
+    TURN_ON_SERVICE,
+)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_ICON): cv.string,
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
```

Comment:
This is legacy and shouldn't be needed anymore.  Pass the domain in the line below